travis/samples/gradle-ng1-mysql/.yo-rc.json

```diff
@@ -22,7 +22,7 @@
       "gatling",
       "protractor"
     ],
-    "jhiPrefix": "custom",
```

Comment:
why is this changed? it should work with custom as well

Reply:
The correct fix is to use the jhiPrefix variable instead of hardcoded `jhi` prefix so that it will use what is set properly

Reply:
sorry - my fault - i will revert this. Thank you for your review.

Reply:
I reverted this back to "custom" - didn´t see your last advise - can you add the variable - or give me a hint about the syntax to access the variable ?
Thanks!